t/plugin_tests/fcrdns

```diff
@@ -0,0 +1,60 @@
+#!perl -w
+
+use strict;
+#use POSIX qw(strftime);
+
+#use Qpsmtpd::Address;
+use Qpsmtpd::Constants;
+
+my $test_email = 'matt@tnpi.net';
+
+sub register_tests {
+    my $self = shift;
+
+    $self->register_test('_is_valid_localhost');
+    $self->register_test('_is_fqdn');
+}
+
+
+sub _is_valid_localhost {
+    my $self = shift;
+    my @passes = qw/ 127.0.0.1 127.1.1.1 127.255.255.255 /;
+    my @fails  = qw/ 128.0.0.1 126.1.1.1 /;
+
+    foreach my $pass (@passes) {
+        $self->qp->connection->remote_ip($pass);
+        cmp_ok( $self->is_valid_localhost(), '==', 1, "$pass, true");
```

Comment:
petty, wouldn't `is( ..., 1 )` be just as well as `cmp_ok( ..., '==', 1 )` here, but a bit less to read?